src/index.ts

```diff
@@ -139,6 +140,10 @@ app.post("/webhook", async (c) => {
   }
 });
 
+app.get("/", (c) => {
+  return c.json({ message: "Hello, world!" });
+});
```

Comment:
No error handling on this express route? That's kinda mid bestie 😭, wrap it up in try-catch to handle potential errors and not leave it hanging.